combat-trainer.lic

```diff
@@ -1352,7 +1352,11 @@ class SpellProcess
 
   def cast_ritual(data, game_state)
     if summoned = game_state.summoned_info(game_state.weapon_skill)
-      break_summoned_weapon(game_state.weapon_name)
+      if DRStats.moon_mage?
+        bput('wear moon', 'telekinetic')
```

Comment:
❓ Does line 1371 need a conditional to 'get' the moonblade?

Reply:
Yes it does

Reply:
It also looks like a get and wield moonblade command might be good targets for common methods in combat-trainer as I see they are used extensively

Reply:
Thoughts on creating common-moonmage.lic? This could also house astrology functions. Combat astrology could use methods commonized from astrology.lic

Reply:
Sounds like a good idea 👍